packages/terra-functional-testing/src/commands/utils/GitHubRepo.js

```diff
@@ -0,0 +1,150 @@
+/**
+ * This module provides a simplified interface to github repos.
+ *
+ * It's basically a wrapper around octokit. The purpose here is to avoid people
+ * re-implememnting octokit w/ auth and error handling in many places.
+ *
+ * The JSON datatypes are filtered versions of the JSON representations you
+ * can read about at https://docs.github.com/en/rest.
+ */
+
+const { Octokit } = require('octokit');
+
+/**
+ * Information about a PR.
+ * @typedef {Object} PullRequest
+ * @property {number} number
+ * @property {Object} head - The head branch
+ * @property {string} head.ref
+ * @property {Object} base - The base branch
+ * @property {string} base.ref
+ *
+ * @see {@link https://git-scm.com/book/en/v2/Git-Internals-Git-References|Git References} to learn about refs.
+ */
+
+/**
+ * Create a simplified view of a PR.
+ *
+ * @param {object} json A PR from the ReST API.
```

Comment:
I think Object is should be capitalized according to https://jsdoc.app/tags-param.html
```suggestion
 * @param {Object} json A PR from the ReST API.
```